specification/paths/ValidateCredentials.json

```diff
@@ -0,0 +1,50 @@
+{
+  "get": {
+    "tags": [
+      "ValidateCredentials"
+    ],
+    "summary": "Validate credentials.",
+    "description": "This endpoint validates the credentials in the headers with the system of the carrier.",
+    "responses": {
+      "200": {
+        "description": "The credentials are valid.",
+        "schema": {
+          "type": "object",
+          "required": [
+            "valid"
+          ],
+          "additionalProperties": false,
+          "properties": {
```

Comment:
I think that for consistency it's better to wrap the properties in a `data` object.